docs/systemd-service.md

```diff
@@ -0,0 +1,21 @@
+# Run flagD as a systemd Service
```

Comment:
I think we might want at least a link to this from the main doc.

Reply:
Done